### PR TITLE
Corrected TestFixtureSourceAttribute Summary Description

### PR DESCRIPTION
**Pull request by sboptimal**
Changed summary description to match class name definition

### CODE REVIEW

File: src/NUnitFramework/framework/Attributes/TestFixtureSourceAttribute.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// TestCaseSourceAttribute indicates the source to be used to
+    /// TestFixtureSourceAttribute indicates the source to be used to
```
Comment (jnm2) [inline src/NUnitFramework/framework/Attributes/TestFixtureSourceAttribute.cs]: Hey, would you mind doing `<see cref=` while you're here? Thanks for doing this!

**Comment by sboptimal** [comment]
Added cref as requested by @jmn2.  I think this is what you are after.  If not let me know and I will correct.

**Comment by sboptimal** [comment]
Thanks.  Didn't see them until I had already committed.

**Comment by mikkelbu** [comment]
No problem 😄 . And thanks for the contribution

**Comment by jnm2** [comment]
Perfect! Appreciate you catching and fixing this!

**Pull request merged**